app/albums/[id]/page.tsx

```
@@ -1,115 +1,53 @@
-	// 			console.log("Fetchに失敗しました: ", error);
-	// 			setLoading(false);
-	// 		}
-	// 	};
-
-	// 	fetchData();
-	// }, []);
+	const albumId = params.id;
+	const [photos, setPhotos] = useState<string[]>([]);
+	const [loading, setLoading] = useState(true);
+
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: albumId

This dependency is not specified in the hook dependency list.

Either include it or remove the dependency array

(lint/correctness/useExhaustiveDependencies)